recurly/base_client.py

```diff
@@ -76,3 +76,19 @@ def _interpolate_path(self, path, *args):
         self._validate_path_parameters(args)
 
         return path % tuple(map(lambda arg: urllib.parse.quote(arg, safe=""), args))
+
+    def _url_encode(self, params):
+        """Encode query params for URL. We need to customize this to conform to Recurly's API"""
+        r_params = {}
+
+        for k, v in params.items():
+            # join lists w/ a comma (CSV encoding)
+            if isinstance(v, list) or isinstance(v, tuple):
+                r_params[k] = ",".join(v)
+            # booleans need to be downcased
+            elif isinstance(v, bool):
+                r_params[k] = "true" if v else "false"
```

Comment:
What about `r_params[k] = str(v).lower`?

Reply:
Thought about it some: the pro with this approach is there is a little less computation. you don't need to go through each char in the string and apply a test, and using lower() copies the string. The con is that it's a little less idiot-proof but I figure the comment should help us there. Maybe i explicitly add "dont change this" type language?